ocs_ci/ocs/rados_utils.py

```diff
@@ -267,3 +269,55 @@ def get_mgr_proxy_container(self, node, docker_image, proxy_container="mgr_proxy
             ][0]
 
         return mgr_object
+
+
+def corrupt_pg(osd_deployment, pool_name, pool_object):
+    """
+    Rewrite given object in a ceph pool with /etc/shadow file.
+
+    Args:
+        osd_deployment (object): OSD deployment object where PG will be corrupted
+        pool_name (str): name of ceph pool to be corrupted
+        pool_object (str): name of object to be corrupted
+    """
+    osd_pod = osd_deployment.pods[0]
+    osd_data = osd_pod.get()
+    osd_containers = osd_data["spec"]["containers"]
+    original_osd_cmd = " ".join(osd_containers[0].get("command"))
+    original_osd_args = osd_containers[0].get("args")
+    original_osd_args = [",".join(arg.split()) for arg in original_osd_args]
+    original_osd_args.remove("--foreground")
+    original_osd_args = " ".join(original_osd_args)
```

Comment:
Maybe we can add a log message like: `logger.info(f"original_osd_args = {original_osd_args}")`